e2e2/test/images/neuron/Dockerfile

```diff
@@ -1,16 +1,12 @@
 FROM public.ecr.aws/docker/library/ubuntu:20.04
 
-# Neuron SDK components version numbers
-ARG NEURONX_DISTRIBUTED_VERSION=0.7.0
-ARG NEURONX_CC_VERSION=2.13.72.0
-ARG NEURONX_FRAMEWORK_VERSION=2.1.2.2.1.0
-ARG NEURONX_COLLECTIVES_LIB_VERSION=2.20.22.0-c101c322e
-ARG NEURONX_RUNTIME_LIB_VERSION=2.20.22.0-1b3ca6425
-ARG NEURONX_TOOLS_VERSION=2.17.1.0
+# Neuron repos
+ARG APT_REPO=https://apt.repos.neuron.amazonaws.com
+ARG PIP_REPO=https://pip.repos.neuron.amazonaws.com
```

Comment:
Are these maintained by the Neuron team or something you created?

Reply:
It is maintained by the Neuron team
https://awsdocs-neuron.readthedocs-hosted.com/en/latest/containers/docker-example/inference/Dockerfile-inference-dlc.html?highlight=https%3A%2F%2Fapt.repos.neuron.amazonaws.com#dlc-sample-dockerfile-for-application-container

Reply:
Ok cool. Again, just a general comment/callout... Maybe we'll want to manage these on our own at some point, but for now this totally makes sense to me.